### PR TITLE
Use Minitest instead of Test::Unit

### PR DESCRIPTION
**Pull request by frodsan**
Related to https://github.com/puma/puma/issues/1151

### CODE REVIEW

**Comment by frodsan** [comment]
There is one test file that doesn't pass in isolation:

```
$ ruby -Ilib -Itest test/test_binder.rb
Run options: --seed 48927

# Running:

E* Listening on tcp://localhost:10001
.


Fabulous run in 0.005208s, 384.0246 runs/s, 192.0123 assertions/s.

  1) Error:
TestBinder#test_localhost_addresses_dont_alter_listeners_for_ssl_addresses:
NoMethodError: undefined method `check' for Puma::MiniSSL:Module
    /Users/frodsan/Code/frodsan/puma/lib/puma/binder.rb:149:in `block in parse'
    /Users/frodsan/Code/frodsan/puma/lib/puma/binder.rb:88:in `each'
    /Users/frodsan/Code/frodsan/puma/lib/puma/binder.rb:88:in `parse'
    test/test_binder.rb:27:in `test_localhost_addresses_dont_alter_listeners_for_ssl_addresses'

2 runs, 1 assertions, 0 failures, 1 errors, 0 skips
```

I'm investigating why, maybe a require is missing.

**Comment by nateberkopec** [comment]
LGTM

File: test/test_tcp_logger.rb [hunk 1]
```diff
@@ -1,10 +1,7 @@
-require 'minitest/autorun'
-require 'minitest/assertions'
+require_relative "test_helper"
```
Comment (nateberkopec) [inline test/test_tcp_logger.rb]: Why does this one have to be `require_relative`?

Comment (frodsan) [inline test/test_tcp_logger.rb]: Oops ... fixed :)

**Comment by nateberkopec** [comment]
Praise be. Hopefully this shouldn't affect outstanding PRs too much.

**Comment by frodsan** [comment]
👍  I hope not

**Comment by nateberkopec** [comment]
just checked, doesnt look like any are affected

**Pull request merged**